### PR TITLE
Notifications tab + new hierarchy

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Reorganize menu, new hierarchy, separates tabs. Was @Quetzacoalt91 work, just add refactoring to prepare new categories.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | #10050 #10049 
| How to test?  | Follow tickets instructions

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9454)

### CODE REVIEW

**Review by tomlev — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9454)* status: 0 of 24 files reviewed, 2 unresolved discussions (waiting on @PierreRambaud and @Quetzacoalt91)

___
*[install-dev/upgrade/php/ps_1750_update_module_tabs.php, line 37 at r2](https://reviewable.io/reviews/prestashop/prestashop/9454#-LK27w4g7QogSKovA1wD:-LK27w4hEOt96lTdIVgP:bic4ren) ([raw file](https://github.com/prestashop/prestashop/blob/e455e21bc71e9a1ec6ac35c4ff72660f4c953dad/install-dev/upgrade/php/ps_1750_update_module_tabs.php#L37)):*
> ```PHP
>     );
> 
>     include_once('add_new_tab.php');
> ```

parenthesis not needed here
___
*[src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php, line 686 at r2](https://reviewable.io/reviews/prestashop/prestashop/9454#-LK2Cmsj6QAJnbIm0dA7:-LK2Cmsj6QAJnbIm0dA8:b3kre) ([raw file](https://github.com/prestashop/prestashop/blob/e455e21bc71e9a1ec6ac35c4ff72660f4c953dad/src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php#L686)):*
> ```PHP
> 
>     /**
>      * @param TODO
> ```

?

**Review by PierreRambaud — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9454)* status: 0 of 24 files reviewed, 2 unresolved discussions (waiting on @tomlev and @Quetzacoalt91)

___
*[install-dev/upgrade/php/ps_1750_update_module_tabs.php, line 37 at r2](https://reviewable.io/reviews/prestashop/prestashop/9454#-LK27w4g7QogSKovA1wD:-LK5oM445Q4Mk4w2YcLN:b-896fix) ([raw file](https://github.com/prestashop/prestashop/blob/e455e21bc71e9a1ec6ac35c4ff72660f4c953dad/install-dev/upgrade/php/ps_1750_update_module_tabs.php#L37)):*
<details><summary><i>Previously, tomlev (Thomas LEVIANDIER) wrote…</i></summary><blockquote>

parenthesis not needed here
</blockquote></details>

Done.
___
*[src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php, line 686 at r2](https://reviewable.io/reviews/prestashop/prestashop/9454#-LK2Cmsj6QAJnbIm0dA7:-LK5oMyX3X1HMxA7ski8:b-896fix) ([raw file](https://github.com/prestashop/prestashop/blob/e455e21bc71e9a1ec6ac35c4ff72660f4c953dad/src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php#L686)):*
<details><summary><i>Previously, tomlev (Thomas LEVIANDIER) wrote…</i></summary><blockquote>

?
</blockquote></details>

Done.

**Review by tomlev — DISMISSED** [review]
> *[Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9454)* status: 0 of 24 files reviewed, all discussions resolved (waiting on @Quetzacoalt91)

**Review by Quetzacoalt91 — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
@PierreRambaud can you have a look at the travis log? There's an issue with a method parameters.

**Comment by PierreRambaud** [comment]
Currently checking, same for @mickaelandrieu PR :/

**Comment by PierreRambaud** [comment]
@Quetzacoalt91 Can you have a review on this? :)

File: src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php [hunk 1]
```diff
@@ -83,7 +85,7 @@ public function catalogAction()
     /**
      * Controller responsible for displaying "Catalog Module Grid" section of Module management pages with ajax.
      *
-     * @AdminSecurity("is_granted(['read', 'create'], 'ADMINMODULESSF_')")
+     * @AdminSecurity("is_granted(['read', 'create', 'update', 'delete'], 'ADMINMODULESSF_')")
```
Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php]: Why do we ask for all privileges to reach the catalog content? Modules actions should are allowed accordingly.

Comment (PierreRambaud) [reply src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php]: You can't delete something you can't see :D
This is how it's configure in many Symfony controller, just did the same thing :/

**Review by Quetzacoalt91 — DISMISSED** [review]

**Comment by marionf** [comment]
@PierreRambaud & @Quetzacoalt91 

In [This ticket](http://forge.prestashop.com/browse/FF-176), it seems Installed modules should be rename in Modules and Selection in Module catalog. Currently, it's not.

Also, in Alerts and Updates tabs, modules are not sorted by categories available on addons

capture d ecran_205


capture d ecran_206

**Comment by Quetzacoalt91** [comment]
@marionf The categories are not part of this PR. This is another issue to be done: https://github.com/PrestaShop/PrestaShop/issues/10048

**Comment by PierreRambaud** [comment]
@marionf @Quetzacoalt91 Seen with Lou, Menu is ok now =)

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
> @marionf @Quetzacoalt91 Seen with Lou, Menu is ok now =)

so LGTM?

**Pull request merged**